packages/simcore-sdk/src/simcore_sdk/node_ports_v2/serialization_v2.py

```diff
@@ -0,0 +1,86 @@
+import json
+from typing import Any, Dict, Set
+
+from aiopg.sa.result import RowProxy
+
+from ..node_ports.dbmanager import DBManager
+from ..node_ports.exceptions import InvalidProtocolError
+from .nodeports_v2 import Nodeports
+
+NODE_REQUIRED_KEYS: Set[str] = {
+    "schema",
+    "inputs",
+    "outputs",
+}
+
```

Comment:
MINOR: why don't we name all serialization/deserialization operations like ``dump`` and ``load``. All other libraries like ``pyyaml, json, pickle, ...`` do so. and could even use the same interface??

Reply:
done